module/jobqueue/types.go

```diff
@@ -0,0 +1,9 @@
+package jobqueue
+
+import (
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/storage"
+)
+
+type Job = storage.Job
+type JobID = module.JobID
```

Comment:
Why the type alias?

Reply:
Initially I haven't decided where to put the type definition of the `Job` and `JobID`, sometimes, it causes cycular dependency. So I made an alias, just so it's easy to move the definition around.